### DISCUSSION TITLE
connect to https

### DISCUSSION THREAD

**Discussion by rnaidabdellah** [jesper-raemaekers/python-polarion#184] [General]
Hi All,

I tried to use this library to connect to our polarion server that is using https protocol.

but all options of connection lead to this kind of error:
SSLError: ("bad handshake: Error([('SSL routines', 'tls_process_server_certificate', 'certificate verify failed')])",)

Any idea how to solve this issue?

Thanks in advance,

**Comment by Polarator** [comment]
Your Python installation needs access to the installed certificates. 
For us the installation of the library "pip-system-certs" helped.
